mmcv/ops/point_sample.py

```diff
@@ -1,9 +1,80 @@
 # Modified from https://github.com/facebookresearch/detectron2/tree/master/projects/PointRend  # noqa
 
+from os import path as osp
+
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 from torch.nn.modules.utils import _pair
+from torch.onnx.operators import shape_as_tensor
+
+from mmcv.ops import get_onnxruntime_op_path
+
+
+def bilinear_grid_sample(im, grid, align_corners=False):
```

Comment:
We could add docstring for the function `bilinear_grid_sample`

Reply:
Added.